meteor/lib/collections/TriggeredActions.ts

```diff
@@ -0,0 +1,40 @@
+import { Meteor } from 'meteor/meteor'
+import { IBlueprintTriggeredActions, SomeBlueprintTrigger } from '@sofie-automation/blueprints-integration'
+import { registerCollection, ProtectedString, ProtectedStringProperties } from '../lib'
+import { ShowStyleBaseId } from './ShowStyleBases'
+import { ObserveChangesForHash, createMongoCollection } from './lib'
+import { registerIndex } from '../database'
+import { PeripheralDeviceId } from './PeripheralDevices'
+
+/** A string, identifying a ShowStyleVariant */
+export type TriggeredActionId = ProtectedString<'TriggeredActionId'>
+
+export type DBBlueprintTrigger = SomeBlueprintTrigger & {
+	deviceId?: PeripheralDeviceId
+}
+
+export interface DBTriggeredActions extends ProtectedStringProperties<IBlueprintTriggeredActions, '_id'> {
+	_id: TriggeredActionId
+	/** Id of parent ShowStyleBase. If null, this is a system-wide triggered action */
+	showStyleBaseId: ShowStyleBaseId | null
+
+	/** Triggers, with attached device info alongside */
+	triggers: DBBlueprintTrigger[]
+
+	_rundownVersionHash: string
+}
+
+export type TriggeredActionsObj = DBTriggeredActions
+export const TriggeredActions = createMongoCollection<TriggeredActionsObj, DBTriggeredActions>('triggeredActions')
+registerCollection('TriggeredActions', TriggeredActions)
+
+registerIndex(TriggeredActions, {
+	showStyleBaseId: 1,
+	_rank: 1,
```

Comment:
The `_rank` isnt really needed to be added to the index, since it isn't used anywhere server-side (client-side queries that use $sort doesn't use the index).

Reply:
The idea to add _rank here was that, once server-side triggers are implemented, this would be useful for the server-side to sort them, but I guess it can be removed for now.

Reply:
Ah ok! Makes sense!